hack/dev/kind/gitea/deploy.py

```diff
@@ -18,6 +18,7 @@
 GITEA_NS = os.environ.get("GITEA_NS", "gitea")
 GITEA_REPO_NAME_E2E = os.environ.get("GITEA_REPO_NAME", "pac-e2e")
 GITEA_REPO_NAME_PERSO = os.environ.get("GITEA_REPO_NAME_PERSO", "pac")
+OPENSHIFT_ROUTE_FORCE_HTTP = os.environ.get("OPENSHIFT_ROUTE_FORCE_HTTP", False)
```

Comment:
Can we add this env info in README ?

Reply:
oops just saw this, yeah we can add them :)